platform/broadcom/saibcm-modules/debian/changelog

```diff
@@ -1,3 +1,9 @@
+opennsl (6.5.21) unstable; urgency=medium
```

Comment:
6.5.21? not 4.3.1.10?

Reply:
These files are copied from HSDK-6.5.21 and not really related to BCMSAI version. Do you think this should be same as BCMSAI version.

Reply:
the problem is that sometime, they will update these kernel module, even if sdk is not updated.

Reply:
ok. Will update it to reflect BCMSAI version.